src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -48,11 +48,12 @@ public Instant getTimestamp() {
 
     @SuppressWarnings("unused, used from DAO")
     @JsonProperty("item-hash")
+    @JsonIgnore
```

Comment:
This method should be removed after these changes and the database changes are integrated.